cli/src/config/templates.toml

```diff
@@ -39,6 +39,20 @@ log = 'builtin_log_compact'
 op_log = 'builtin_op_log_compact'
 show = 'builtin_log_detailed'
 
+op_log_node = 'if(current_operation, "@", "◉")'
```

Comment:
Do we want to make the "normal" symbol consistent with `log_node`? Do we want to make labels/colors consistent?